### PR TITLE
Script to fetch events from the blockchain and submit them to the sequencer

### PR DESCRIPTION
**Pull request by Dzejkop**
Example usage:
```
node scripts/fetchAndSubmitAddMemberEvents.js \
    --block-span 1000 \
    --rpc-url https://polygon-rpc.com \
     -o events \
     -e events.errors \
     -s http://localhost:8080
```

Options:
```
Options:
      --help               Show help                                   [boolean]
      --version            Show version number                         [boolean]
  -r, --rpc-url            The RPC URL to connect to (supports RPC_URL env var)
                                              [default: "http://localhost:8545"]
  -s, --sequencer-url      The sequencer url to use
  -a, --address            The address of the smart contract to read events from
                         [default: "0xD81dE4BCEf43840a2883e5730d014630eA6b7c4A"]
  -f, --from-block         The block to start reading events from
                                                             [default: 39186981]
  -t, --to-block           If not set, will fetch the current block
  -b, --block-span         The number of blocks to read events from at a time
                                                                [default: 10000]
  -o, --output-file        If set, will write the events to a file
  -e, --error-output-file  If set, will errors that occurred during submission
                           to a file
```

I added the option to save errors and correctly submitted commitments to files - in case something goes wrong during migration.

### CODE REVIEW

File: scripts/fetchAndSubmitAddMemberEvents.js [hunk 1]
```diff
@@ -0,0 +1,147 @@
+import dotenv from 'dotenv'
```
Comment (iamrecursion) [inline scripts/fetchAndSubmitAddMemberEvents.js:1]: Given how critical this migration process is, it's probably worth adding some comments here to explain what it's doing.

File: scripts/fetchAndSubmitAddMemberEvents.js [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+        const identityCommitment = parsed.args[1]
+
+        return identityCommitment.toHexString()
+    })
+
+    return Promise.all(commitments.map(async (commitment) => {
+        try {
+            if (argv.sequencerUrl !== undefined) {
+                await axios.post(`${argv.sequencerUrl}/insertIdentity`, { identityCommitment: commitment })
+            }
```
Comment (iamrecursion) [inline scripts/fetchAndSubmitAddMemberEvents.js:70]: Is `sequencerUrl === undefined` an error condition? If so it should probably bail here, and if it _isn't_ it should at least warn.

**Review by iamrecursion — APPROVED** [review]
Good stuff. Just a couple of minor things.

File: scripts/fetchAndSubmitAddMemberEvents.js (hunk 2 above)
Comment (Dzejkop) [reply scripts/fetchAndSubmitAddMemberEvents.js:70]: Not an error condition, I also envision a case where we don't provide sequencerUrl to first get the commitments into a file. But given that I should probably add an option to also read form a file...

File: scripts/fetchAndSubmitAddMemberEvents.js (hunk 1 above)
Comment (Dzejkop) [reply scripts/fetchAndSubmitAddMemberEvents.js:1]: Added some comments

**Review by iamrecursion — APPROVED** [review]

**Review by gswirski — APPROVED** [review]

**Pull request merged**